sklearn/linear_model/tests/test_sgd.py

```diff
@@ -1294,32 +1294,34 @@ def test_large_regularization(penalty):
 
 def test_tol_parameter():
     # Test that the tol parameter behaves as expected
-    X = StandardScaler().fit_transform(iris.data)
```

Comment:
Why are tests of SGD changed in this PR?

Reply:
True. Actually, I've no idea. It should not, the test does not touch anything of the types AFIK.

Reply:
This PR actually adds `SequentialDatset64` to SGD. So I guess its normal to modify a bit the tests. But it does not add 32 support and only re-runs the test with multiple seeds. 

So I'll revert the changes in SGD and alias `SequentialDatset64` to `SequentialDatset` it makes more sense.

Reply:
This was actually added by @NelleV 	and @ogrisel